prow/cmd/tide/main.go

```diff
@@ -171,26 +170,16 @@ func main() {
 	if err != nil {
 		logrus.WithError(err).Fatal("Error constructing mgr.")
 	}
-	// Make sure the manager creates a cache for ProwJobs by requesting an informer
-	if _, err := mgr.GetCache().GetInformer(&prowjobsv1.ProwJob{}); err != nil {
-		logrus.WithError(err).Fatal("Error getting ProwJob informer.")
+	c, err := tide.NewController(githubSync, githubStatus, mgr, cfg, gitClient, o.maxRecordsPerPool, opener, o.historyURI, o.statusURI, nil)
+	if err != nil {
+		logrus.WithError(err).Fatal("Error creating Tide controller.")
 	}
-
 	interrupts.Run(func(ctx context.Context) {
 		if err := mgr.Start(ctx.Done()); err != nil {
 			logrus.WithError(err).Fatal("Mgr failed.")
 		}
+		logrus.Info("Mgr finished gracefully.")
 	})
-	mgrSyncCtx, mgrSyncCtxCancel := context.WithTimeout(context.Background(), 30*time.Second)
-	defer mgrSyncCtxCancel()
-	if synced := mgr.GetCache().WaitForCacheSync(mgrSyncCtx.Done()); !synced {
```

Comment:
Is `WaitForCacheSync` no longer needed for some reason?

Reply:
This was an oversight, re-added it